submissions/bolarinwa_fasanmi/HTML/landingpage.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>landing e-commerce page</title>
+    <link rel="stylesheet" href="style1.css">
+    
+</head>
+
+
+<body>
+    <div class="container">
+        <div class="head">
+            <img src="image/logo.png" alt="logo" id="logo">
+            <input type="text" placeholder="search" id="search">
+            <nav>
+                <ul>
+
+                    <li><a href="#">Cart</a></li>
+
+
+                    <li><a href="#">Sign Up</a></li>
+
+
+                    <li><a href="#">Log In</a></li>
+
+                </ul>
+
+            </nav>
+            </span>

```

Comment:
This **span** closing tag does not have an opening tag. You should remove it if it's not needed.